spec/bowling_spec.rb

```diff
@@ -0,0 +1,45 @@
+require 'bowling'
+
+describe 'A bowling game' do
+
+  before do
+    @game = Bowling.new
+  end
+
+  it 'can make a new game' do
```

Comment:
This is sort of implied with the before do statement now so this test can be deleted